modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/JavascriptClientCodegen.java

```diff
@@ -740,6 +742,24 @@ public void setParameterExampleValue(CodegenParameter p) {
         p.example = example;
     }
 
+    @Override
+    public void setParameterExampleValue(CodegenParameter codegenParameter, Parameter parameter) {
+        Schema schema = parameter.getSchema();
+
+        if (parameter.getExample() != null) {
+            codegenParameter.example = parameter.getExample().toString();
+        } else if (parameter.getExamples() != null && !parameter.getExamples().isEmpty()) {
+            Example example = parameter.getExamples().values().iterator().next();
+            if (example.getValue() != null) {
+                codegenParameter.example = example.getValue().toString();
+            }
+        } else if (schema != null && schema.getExample() != null) {
+            codegenParameter.example = schema.getExample().toString();
+        }
+
+        setParameterExampleValue(codegenParameter);
+    }
```

Comment:
thanks for the PR. if you didn't add this code block, does it work for you?

default codegen seems doing that already: https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java#L2134-L2151

Reply:
The problem in the default codegen is that it has early `return;` statements that skip the call.

Reply:
ah ok. i'll try to review and get that fix